.github/PULL_REQUEST_TEMPLATE/base_template.md

```diff
@@ -0,0 +1,64 @@
+---
+name: Pull request template
+about: Template for interchain security templates
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+
+# Description
+
+Please include a summary of the changes and the related issue. If the the issue was ambiguous try to clarify it in this section.
+
+
+# Linked issues
+
+Closes: `#<issue>`
+
+
+# Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Non-breaking changes
+- [ ] New feature (non-breaking changes adding functionality)
```

Comment:
Maybe this? since new features can be breaking or not breaking and we already reference that in the other bullet point
```suggestion
- [ ] New feature (adding functionality)
```

Reply:
Changed to suggested